src/widgets/totalHrsAndGranteeGraph.js

```diff
@@ -0,0 +1,126 @@
+import { Op } from 'sequelize';
+import moment from 'moment';
+import {
+  ActivityReport, ActivityRecipient, Grant, sequelize,
+} from '../models';
+import { REPORT_STATUSES } from '../constants';
+
+function addOrUpdateResponse(traceIndex, res, xValue, valueToAdd) {
+  const valueToUse = parseFloat(valueToAdd, 10) === 0 ? 0 : parseFloat(valueToAdd, 10);
+
+  if (res[traceIndex].x.includes(xValue)) {
+    // Update existing Y value.
+    const indexToUpdate = res[traceIndex].x.indexOf(xValue);
+    res[traceIndex].y[indexToUpdate] += valueToUse;
+  } else {
+    // Add X value and Update Y value.
+    res[traceIndex].x.push(xValue);
+    res[traceIndex].y.push(valueToUse);
+  }
+}
+
+export default async function totalHrsAndGranteeGraph(scopes, query) {
+  // Get the Date Range.
+  const dateRange = query['startDate.win'];
+  const dates = dateRange.split('-');
+
+  // Parse out Start and End Date.
+  let startDate;
+  let endDate;
+
+  let useDays = true;
+
+  // Check if we have a Start Date.
+  if (dates.length > 0) {
+    // eslint-disable-next-line prefer-destructuring
```

Comment:
Don't care for array destructuring?

Reply:
I can't in good conscious change the code to be less readable because lint tells me to do it.

Is this a deal breaker? Do we have any reasons why we should change it? I'm new to this concept.

Reply:
no I was just curious about the change, not a deal breaker for me at all